swagger2.0/oic.r.softwareupdate.swagger.json

```diff
@@ -0,0 +1,199 @@
+    }
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/softwareupdateResURI" : {
+      "get": {
+        "description": "The resource to do scheduled update",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+            "200": {
+              "description" : "",
+              "x-example":
+                {
+                  "rt":   ["oic.r.softwareupdate"],
+                  "purl":  "https://myvendor/myexampleurl",
+                  "swupdateaction": "idle",
+                  "swupdatestate" : "idle",
+                  "result" : 0,
+                  "lastupdate" : "2015-01-09T14:30Z",
+                  "signage" : "vendor",
+                  "updatetime" : "2015-01-09T14:30Z"
+                }
+                ,
+              "schema": { "$ref": "#/definitions/swupdate" }
+            }
+        }
+      },
+      "post": {
+        "description": "Start/Stop collecting and reset the networking monitor resource\n",
+        "parameters": [
+          {"$ref": "#/parameters/interface-rw"},
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/swupdate-update" },
+            "x-example":
+              {
+                  "purl":  "https://myvendor/newversion",
+                  "swupdateaction": "upgrade",
+                  "updatetime" : "2030-01-09T14:30Z"
+                }
+          }
+        ],
+        "responses": {
+            "200": {
+              "description" : "",
+              "x-example":
+                {
+                  "rt":   ["oic.r.softwareupdate"],
+                  "purl":  "https://myvendor/myexampleurl",
+                  "swupdateaction": "upgrade",
+                  "swupdatestate" : "idle",
+                  "result" : 0,
+                  "lastupdate" : "2015-01-09T14:30Z",
+                  "signage" : "vendor",
+                  "updatetime" : "2030-01-09T14:30Z"
+                },
+              "schema": { "$ref": "#/definitions/swupdate" }
+            }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface-rw" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.rw"]
+    },
+    "interface-all" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.rw", "oic.if.baseline"]
+    }
+  },
+  "definitions": {
+    "swupdate" : {
+      "properties": {
+        "rt" : {
+          "items": {
+            "enum": [
+              "oic.r.softwareupdate"
+            ],
+            "type": "string"
+          },
+          "maxItems": 2,
+          "minItems": 2,
```

Comment:
There is only one "rt" value.  Why does this say minItems 2?

Reply:
fixed, max&min = 1